### PR TITLE
449 one pager content updates

### PR DESCRIPTION
**Pull request by jonathanbobel**
Updates Meghan found I missed

### CODE REVIEW

File: app/templates/views/security.html [hunk 1]
```diff
@@ -76,11 +76,6 @@ <h3 class="font-body-lg">Two-factor authentication</h3>
   </ul>
   <p>If signing in with a text message is a problem for your team, <a class="usa-link" href="{{ url_for('main.support') }}">contact us</a> to find out about using an email link instead.</p>
 
-  <img src="{{ asset_url('images/product/security-review-message.png') }}"
-    alt="Screenshot of a teat message in review with the link to 'hide personalization after sending' circled.">
```
Comment (terrazoon) [inline app/templates/views/security.html:80]: typo for test

Comment (ccostino) [reply app/templates/views/security.html:80]: Ah, thankfully this is a deleted line!  Thanks for catching though, @terrazoon!

**Review by ccostino — APPROVED** [review]
Thanks, @jonathanbobel!

**Comment by ccostino** [comment]
@terrazoon you'll need to approve again here since you requested changes originally.  Thanks!

**Review by terrazoon — APPROVED** [review]

File: app/templates/views/security.html [hunk 2]
```diff
@@ -65,9 +65,9 @@ <h2 id="technical-security">Technical security</h2>
   <h3 class="font-body-lg">Protect sensitive information</h3>
   <p>Some messages include sensitive information like security codes or password reset links.</p>
   <p>If you’re sending a message with sensitive information, you can choose to hide those details on the Notify dashboard once the message has been sent. This means that only the message recipient will be able to see that information.</p>
+  <img src="{{ asset_url('images/product/security-review-message.png') }}"
+  alt="Screenshot of a teat message in review with the link to 'hide personalization after sending' circled.">
```
Comment (ccostino) [inline app/templates/views/security.html:69]: Fixing typo:

```suggestion
  alt="Screenshot of a test message in review with the link to 'hide personalization after sending' circled.">
```

**Pull request merged**